decidim-participatory_processes/app/controllers/decidim/participatory_processes/participatory_processes_controller.rb

```diff
@@ -29,7 +29,11 @@ def show
       end
 
       def all_metrics
-        enforce_permission_to :read, :process, process: current_participatory_space
+        if current_participatory_space.show_statistics
+          enforce_permission_to :read, :process, process: current_participatory_space
```

Comment:
Is this related to "Do not expose process statistics in the API if hidden"? Shouldn't we also control decidim-core/app/types/decidim/core/metric_type.rb?

Reply:
Done!